evalml/exceptions/exceptions.py

```diff
@@ -87,3 +87,11 @@ class ObjectiveCreationError(Exception):
 
 class NoPositiveLabelException(Exception):
     """Exception when a particular classification label for the 'positive' class cannot be found in the column index or unique values"""
+
+
+class ParameterNotUsedWarning(UserWarning):
```

Comment:
It shows up on the documentation automatically! https://feature-labs-inc-evalml--2564.com.readthedocs.build/en/2564/autoapi/evalml/exceptions/index.html#evalml.exceptions.ParameterNotUsedWarning

![image](https://media.giphy.com/media/P7JmDW7IkB7TW/giphy.gif)